src/EventStore/Middleware/index.js

```diff
@@ -0,0 +1,248 @@
+
+import {keys, pick, omit, flatten, difference, extend} from 'lodash'
+import { web3 } from '../../env'
+
+var {
+    SolidityEventSchema
+} = require('../EventTypes')
+
+var {
+    eventsFromTransaction
+} = require('../Transactions')
+
+
+const solidityEventProperties = keys(SolidityEventSchema)
```

Comment:
This naming is really misleading with the fact that we also have a SolidityEventPropertySchema.